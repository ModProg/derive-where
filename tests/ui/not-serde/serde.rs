use std::marker::PhantomData;

use derive_where::derive_where;

#[derive_where(Clone)]
#[serde(crate = "serde_")]
struct InvalidAttribute<T>(PhantomData<T>);

fn main() {}
