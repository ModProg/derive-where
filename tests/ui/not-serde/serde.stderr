error: cannot find attribute `serde` in this scope
 --> tests/ui/not-serde/serde.rs:6:3
  |
6 | #[serde(crate = "serde_")]
  |   ^^^^^
